Makefile

```diff
@@ -0,0 +1,44 @@
+# Define the dependencies
+DEPENDENCIES = chafa convert jq curl
+
+# Define the installation directories
+INSTALL_DIR = /usr/local
+
+# Installation instructions
+install: check_dependencies
+	@echo "\033[1;32m Installing... \033[0m"
+	sudo mkdir -pv $(INSTALL_DIR)
```

Comment:
Are `sudo` commands necessary inside the `Makefile`? I believe these should be left out and if the `INSTALL_DIR` requires admin privileges then the user can call `sudo make install/uninstall`, right?